MAUI/Cartesian-Charts/Axis/Ticklines.md

```diff
@@ -50,6 +50,42 @@ chart.YAxes.Add(secondaryAxis);
 
 {% endtabs %}
 
+## Positioning the ticklines
+
+The positioning of tick lines inside or outside the chart area can be controlled using the [TickPosition]() property. By default, the tick lines are positioned `AxisElementPosition.Outside` the chart area. 
```

Comment:
Improve the default value like follows -https://help.syncfusion.com/maui/cartesian-charts/axis/axislabels#edge-labels-drawing-mode

Reply:
Updated the content as present in the link